CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# v2.1.0 (2024-07-08)
+
+## OS Changes
+* Update kernels: 5.10.218-208, 5.10.219, 5.15.160-104, and 6.1.94 ([#13], [#17])
+* Include kmod-6.1-neuron package in core kit ([#21])
+* Enable host containers and in-place updates to be optional ([#23])
```

Comment:
You should add a line for your PR here:

```suggestion
* Enable host containers and in-place updates to be optional ([#23])
* Provide SSM agent as a system service
```

Reply:
Is it okay to put two lines in different categories for a single PR?